netcdfrw2.f

```diff
@@ -5249,7 +5264,13 @@ subroutine f4dwrite
       use tdfinterp_mod, only : tdfinterp
       use zcunix_mod, only : terp1
       use zcunix_mod, only : terp2
-      implicit integer (i-n), real*8 (a-h,o-z)
+      implicit none
+      integer :: j,k,istat1,istat2,istat3,istat4,istat5,istat6,istat7 !local
+      integer :: ir,iz,iv,it ! local
+      real*8 dr,dz_,dv,dt,avg ! local scalars
```

Comment:
im not even sure real*8 is legit anymore, though I do understand it was really common.